.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,14 +1,18 @@
 ---
 name: Bug Report
 about: If things aren't working as expected.
+title: ''
+labels: ''
```

Comment:
Can we automatically add the `bug` label?

Reply:
I personally look at issues that have no labels as a way of them needing my acknowledgment when I go through all the issues. But if it makes it easier for everyone else I am happy to add it.

Apparently, the format is the following:

```
labels: bug, feedback
```

So for bug issues we would add `bug` label and for feature requests `enhancement` and guessing for question well `question` :) So if the above SGTY I can add them unless you had different labels in mind?